<ui version="4.0" >
 <class>RootDataListBase</class>
 <widget class="QDialog" name="RootDataListBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ROOT-data list</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>11</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="moduleLabel" >
       <property name="text" >
        <string>Data for module ...</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="TextLabel1_2" >
       <property name="text" >
        <string>Comments:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Q3MultiLineEdit" native="1" name="Comments" >
       <property name="maximumSize" >
        <size>
         <width>32767</width>
         <height>100</height>
        </size>
       </property>
       <property name="readOnly" stdset="0" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Q3ListView" name="FileList" >
       <property name="selectionMode" >
        <enum>Q3ListView::Multi</enum>
       </property>
       <column>
        <property name="text" >
         <string>Available data sets                    </string>
        </property>
        <property name="clickable" >
         <bool>true</bool>
        </property>
        <property name="resizable" >
         <bool>true</bool>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Time stamp                          </string>
        </property>
        <property name="clickable" >
         <bool>true</bool>
        </property>
        <property name="resizable" >
         <bool>true</bool>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>ID</string>
        </property>
        <property name="clickable" >
         <bool>false</bool>
        </property>
        <property name="resizable" >
         <bool>false</bool>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="AllButton" >
         <property name="text" >
          <string>Select all</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Spacer2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="DeAllButton" >
         <property name="text" >
          <string>De-select all</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="OKButton" >
         <property name="text" >
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Spacer3" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="AbortButton" >
         <property name="text" >
          <string>Abort</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="TextLabel1" >
     <property name="text" >
      <string/>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ListView</class>
   <extends>Q3Frame</extends>
   <header>q3listview.h</header>
  </customwidget>
  <customwidget>
   <class>Q3MultiLineEdit</class>
   <extends>QWidget</extends>
   <header>q3multilineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
